src/views/Home/components/NewsCard/components/CategorizedNews/index.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import Collapse from '@material-ui/core/Collapse';
+import { Button } from '@material-ui/core';
+import { gordonColors } from '../../../../../../theme';
+import NewsService from '../../../../../../services/news';
+import NewsItem from '../NewsItem';
+
+//Written by Jessica Guan
+
+export default class CategorizedNews extends Component {
+  constructor(props) {
+    super(props);
+
+    this.handleExpandClick = this.handleExpandClick.bind(this);
+
+    this.state = {
+      todaysNews: [],
+
+      open: false,
+    };
+  }
+
+  componentWillMount() {
+    this.loadNews();
+  }
+
+  // loads the current day's news for this category
+  async loadNews() {
+    let todaysNews;
+
+    //this doesn't do anything and may break since the backend
+    //isn't written
+    todaysNews = await NewsService.getTodaysNews(this.props.categoryID);
```

Comment:
If the back end is not written and it's using a async function, it probably will actually crash the site because it will never get anything back. I'd recommend commenting out the fetch you do in the todaysNews const.